### DISCUSSION TITLE
Achievements for past contributions

### DISCUSSION THREAD

**Discussion by david-mcafee** [community/community#20836] [Other Feature Feedback, Questions, & Ideas]
It would be nice if, at some point, achievements applied to contributions that were made prior to the existence of achievements. Thanks for considering!

**Comment by radar-blip** [comment]
I wouldn't hold my breath on that one. lol

**Comment by jfullstackdev** [comment]
we don't know or maybe another awesome badge is waiting, the catch is just be active and contribute,

I received the Arctic Badge without really expecting it, that was really an awesome surprise, and we don't know whether it will be given again or maybe another version of it
